File: rb/test/exists_test.rb
# CurrencyExchange SDK exists test

require "minitest/autorun"
require_relative "../CurrencyExchange_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = CurrencyExchangeSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- CurrencyExchange SDK exists test

local sdk = require("currency-exchange_sdk")

describe("CurrencyExchangeSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
